docs/cellxgene_census_FAQ.md

```diff
@@ -0,0 +1,109 @@
+# Frequently Asked Questions for CZ CELLxGENE Discover Census
+
+Last updated: Apr, 2023.
+
+- [Why should I use the Census?](#Why-should-I-use-the-Census)
+- [Why does the Census not have a normalized layer or embeddings?](#Why-does-the-Census-not-have-a-normalized-layer-or-embeddings)
+- [How does the Census differentiate from other services?](#How-does-the-Census-differentiate-from-other-services?)
+- [Can I query human and mouse data in a single query?](#Can-I-query-human-and-mouse-data-in-a-single-query)
+- [Where are the Census data hosted?](#Where-are-the-Census-data-hosted)
+- [How can I increase the performance of my queries?](#How-can-I-increase-the-performance-of-my-queries)
+- [Can I use conda to install the Census Python API?](#Can-I-use-conda-to-install-the-Census-Python-API)
+- [How can I ask for support?](#How-can-I-ask-for-support)
+- [How can I ask for new features?](#How-can-I-ask-for-new-features)
+- [How can I contribute my data to the Census?](#How-can-I-contribute-my-data-to-the-Census)
+- [Why do I get an `ArraySchema` error when opening the Census?](#Why-do-I-get-an-ArraySchema-error-when-opening-the-Census)
+
+## Why should I use the Census?
+
+The Census provides efficient low-latency access via Python and R APIs to most single-cell RNA data from [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/). 
+
+To accelerate your computational research, **you should use the Census**:
```

Comment:
I would include the "if you want" or "if you want to" on line 21 rather than repeating it in all the bullet points below.